client/src/pages/Register/index.jsx

```diff
@@ -15,7 +15,8 @@ const SignupForms = () => {
 	const [firstStepSuccess, setFirstStepSucess] = useState(false);
 	const [secondStepSuccess, setSecondStepSucess] = useState(false);
 	const [firstStepValues, setFirstStepValues] = useState({ name: '', password: '', email: '' });
-	const [secondStepValues, setSecondStepValues] = useState({ type: personTypes[0].value, aboutme: '', address: '' });
```

Comment:
why to take out the address? also, if you take out of this function you have to fix line 54 in this file